src/CommunityToolkit.Maui/Converters/ImageResourceConverter.shared.cs

```diff
@@ -9,24 +10,25 @@ namespace CommunityToolkit.Maui.Converters;
 /// </summary>
 public class ImageResourceConverter : IValueConverter
 {
-    /// <summary>
-    /// Converts embedded image resource ID to it ImageSource.
-    /// </summary>
-    /// <param name="value">The value to convert.</param>
-    /// <param name="targetType">The type of the binding target property. This is not implemented.</param>
-    /// <param name="parameter">Additional parameter for the converter to handle. This is not implemented.</param>
-    /// <param name="culture">The culture to use in the converter. This is not implemented.</param>
-    /// <returns>The ImageSource related to the provided resource ID of the embedded image. If it's null it will returns null.</returns>
-    public object? Convert(object? value, Type? targetType, object? parameter, CultureInfo? culture)
-    {
-        if (value == null)
-            return null;
+	/// <summary>
+	/// Converts embedded image resource ID to it ImageSource.
+	/// </summary>
+	/// <param name="value">The value to convert.</param>
+	/// <param name="targetType">The type of the binding target property. This is not implemented.</param>
+	/// <param name="parameter">Additional parameter for the converter to handle. This is not implemented.</param>
+	/// <param name="culture">The culture to use in the converter. This is not implemented.</param>
+	/// <returns>The ImageSource related to the provided resource ID of the embedded image. If it's null it will returns null.</returns>
+	[return: NotNullIfNotNull("value")]
```

Comment:
Just FYI - `[return: NotNullIfNotNull("value")]` means that if `value is not null`, then whatever the method returns also won't be null.

Now that I've added it here, I should go through the other converters and clean them up in a future PR as well 😃

Reply:
Ah, that makes sense.  I followed very closely to the `ByteArrayToImageSourceConverter_Tests` since this is my first foray into this project.

Reply:
Ah - I'm glad you pointed that out! 

Let's consolidate a couple duplicated methods, then! I'll leave the recommendation in another comment.

Reply:
Yeah, I realized the duplicates, but just wasn't sure what pattern was in use for that kind of stuff.